helper/testhelpers/testhelpers.go

```diff
@@ -623,6 +623,10 @@ func GenerateDebugLogs(t testing.T, client *api.Client) chan struct{} {
 	return stopCh
 }
 
+// VerifyRaftPeers verifies that the raft configuration contains a given set of peers.
+// The `expected` argument is a map consisting of the nodeId string (key) and a
+// boolean representing peer existence (value).
+// For example: { "core-0": true, "core-1": false, "core-2": true }
```

Comment:
@akshya96 how does this description look? Does this clarify the helper?

Reply:
Yes, this is good. Thanks

Reply:
I see that this function does not use this boolean value at all. Is that right? Do we have this only because it is easier to delete a key from a map and in case of an array, we need to iterate through it?